orttraining/orttraining/test/python/orttraining_test_ortmodule_fairscale_sharded_optimizer.py

```diff
@@ -0,0 +1,267 @@
+        if epoch == 0:
+            epoch_0_training = total_training_time
+        if args.test_batch_size > 0:
+            test_time, validation_accuracy = test(args, model, rank, loss_fn, test_dataloader)
+            total_test_time += test_time
+
+    print('\n======== Global stats ========')
+    if args.use_ortmodule:
+        estimated_export = 0
+        if args.epochs > 1:
+            estimated_export = epoch_0_training - (total_training_time - epoch_0_training)/(args.epochs-1)
+            print("  Estimated ONNX export took:               {:.4f}s".format(estimated_export))
+        else:
+            print("  Estimated ONNX export took:               Estimate available when epochs > 1 only")
+        print("  Accumulated training without export took: {:.4f}s".format(total_training_time - estimated_export))
+    print("  Accumulated training took:                {:.4f}s".format(total_training_time))
+    print("  Accumulated validation took:              {:.4f}s".format(total_test_time))
+
+
+    dist.destroy_process_group()
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(
+        description="Benchmark the optimizer state sharding, on a typical computer vision workload"
+    )
+    parser.add_argument("--world_size", action="store", default=2, type=int)
+    parser.add_argument("--epochs", action="store", default=10, type=int)
+    parser.add_argument("--batch_size", action="store", default=256, type=int)
+    parser.add_argument('--lr', type=float, default=0.01, metavar='LR',
+                        help='learning rate (default: 0.01)')
+    parser.add_argument('--use_sharded_optimizer', action='store_true', default=False, 
+                        help='use sharded optim')
+    parser.add_argument('--train-steps', type=int, default=-1, metavar='N',
+                        help='number of steps to train. Set -1 to run through whole dataset (default: -1)')
+    parser.add_argument('--view-graphs', action='store_true', default=False,
+                        help='views forward and backward graphs')
+    parser.add_argument('--log-interval', type=int, default=300, metavar='N',
+                        help='how many batches to wait before logging training status (default: 300)')
+    parser.add_argument("--cpu", action="store_true", default=False)
+    parser.add_argument("--use_ortmodule", action="store_true", default=False, help = "use ortmodule")
+    parser.add_argument('--test-batch-size', type=int, default=64, metavar='N',
+                        help='input batch size for testing (default: 64)')
+    parser.add_argument('--no-cuda', action='store_true', default=False,
+                        help='disables CUDA training')
+    parser.add_argument('--seed', type=int, default=42, metavar='S',
+                        help='random seed (default: 42)')
+    parser.add_argument('--data_dir', type=str, default='./mnist',
```

Comment:
```suggestion
    parser.add_argument('--data-dir', type=str, default='./mnist',
```
The argument is passed as `--data-dir` from `orttraining_ortmodule_distributed_tests.py:run_ortmodule_fairscale_sharded_optimizer_tests`

Reply:
addressed this.